.github/workflows/build.yml

```diff
@@ -94,7 +94,7 @@ jobs:
 
   docker:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
ubuntu-22.04 would be better. The latest is not predictable, is just like using dev-master in composer...

Reply:
I would use `latest` on pipelines to ensure it goes automatically on newer versions. For building tools, I believe it could be ok. If we are strict on pinning versions then we need to change elsewhere to ping the apt packages otherwise we can't ensure reproducibility (at the moment we are not pinning any package in the dockerfile nor in the pipelines).
I'm happy to give it another round to pin everything, the only downside is that it'll require more maintenance to keep it up to date.

Reply:
Ok pin please! ;)